Funky.css

```diff
@@ -0,0 +1,73 @@
+#EspenButton {
+    background-color: #222222;
+    color: white;
+    font-weight: bold;
+    text-decoration: none;
+    border-radius: 10%;
+    width: 8%;
+    height: 3rem;
+    box-shadow: 4px 4px 0px 0px rgba(0,0,0,0.65);
+}
+#BryceButton {
+    background-color: #222222;
+    color: white;
+    font-weight: bold;
+    text-decoration: none;
+    border-radius: 10%;
+    width: 8%;
+    height: 3rem;
+    box-shadow: 4px 4px 0px 0px rgba(0,0,0,0.65);
+}
+#FunkyButton {
+    background-color: #663399;
+    color: white;
+    font-weight: bolder;
+    text-decoration: wavy underline black;
+    border-radius: 100%;
+    width: 10%;
+    height: 5rem;
+    box-shadow: 4px 4px 0px 0px rgba(0,0,0,0.65);
+}
+#FunkyArea {
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
+    width: 100%;
+    margin: 1em;
+}
+#FunkyArea2 {
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
+    width: 100%;
+    margin: 1em;
+}

```

Comment:
IIRC stuff like this can be written:
```
#FunkyArea, #FunkyArea2 {
    display: flex;
    justify-content: center;
    flex-direction: row;
    width: 100%;
    margin: 1em;
}
```

Also just FYI while it's fine here -- naming stuff like "Thing1" and "Thing2" is generally frowned upon

Reply:
But you could consolidate a lot of this CSS. Also this is a good use case for CSS classes as well. Instead of giving ids like 1, 2, 3, 4, 5, and so on, make each area the same class, thus they should have the same CSS.
```
.FunkyArea {

}
```
^ This will target every element with that classname. Alternatively, if each element is the same element e.g. `div` you could target the parent:
```
#ParentOfFunkyPage div {

}
```
^ Targets all divs under the parent.

If you find yourself copy-pasting, take a step back and make sure it's needed.